spec/fixtures/responses/whois.corporatedomains.com/status_registered.txt

```diff
@@ -1,68 +1,130 @@
-Domain Name: google.com
-Registry Domain ID: 
-Registrar WHOIS Server: whois.corporatedomains.com
-Registrar URL: www.cscprotectsbrands.com
-Updated Date: 2012-05-16 09:28:56 -0400
-Creation Date: 1992-11-24 00:00:00 -0500
-Registrar Registration Expiration Date: 2013-11-23 00:00:00 -0500
-Registrar: CORPORATE DOMAINS, INC.
-Registrar IANA ID: 299
-Registrar Abuse Contact Email: admin@internationaladmin.com
+
+Whois Server Version 2.0
```

Comment:
Be careful! .COM is a thin server. This initial part is returned by Verisign, it's not returned by CorporateDomains. The fixture must contain only the specific registrar response.